app/views/posts/index.html.erb

```diff
@@ -1,9 +1,33 @@
-<div>
-     <% @posts.each do |post|%>
-    <ul>
-        <li><%= post.title%></li>
-        <li><%= post.text%></li>
-        <li><%= post.comments_counter%></li>
-    </ul>
-    <%end%>
-</div>
\ No newline at end of file
+ <div class=container>  
+   <div class='photo'>
+ <img image_tag ="<%= @user.photo%>", style='height:50px; width: 40px ;margin-bottom:20px;' >
+   </div>
+   <div class='info'>
+      <h2><%=@user.name%>
+     <p><strong>Number of Posts:</strong><span> <%= @user.posts_counter%></span></p>
+   </div>
+   <div>
+      <% @posts.each do |post|%>
+       
+      <div>
+        <a href=<%= "posts/#{post.id}" %>>
+      <h1><%= post.title%></h1>
+      </a>
+          <p><%= post.text%></p>
+      </div>
+        
+      <%end%>
+   </div>
+     <div class="comments">
+          <% if @comments.length>=1 %>
+          <% @comments.each do |comment| %>
+            <div class="comment">
+              <p>Comments:</p>
+              <p><%= comment[:user] %>: <%= comment[:comment] %></p>
+            </div>
+          <% end %>
+          <% else %>
+            <p class="nocomments">No comments yet</p>
+          <% end %>
+        </div>
```

Comment:
- [ ] Please watch the wireframe here in posts as well. We should see under each post a comment and a like counter.
![image](https://github.com/IsmailChangezi/blog-app-new/assets/91486791/ee0535b3-93f2-4cc2-b7cb-783b2673a2a1)
should be like:
![image](https://github.com/IsmailChangezi/blog-app-new/assets/91486791/7835a624-8af5-4c69-a86f-8cd805ea755b)
- [ ] Add a pagination button under the list. You don't need to implement the functionality of it. But you are going to need it when writing the tests.